payloads/v8/webhook.ts

```diff
@@ -49,8 +49,18 @@ export interface APIWebhook {
 	 * The bot/OAuth2 application that created this webhook
 	 */
 	application_id: Snowflake | null;
+	/**
+	 * The guild of the channel that this webhook is following (returned for Channel Follower Webhooks)
+	 */
 	source_guild?: APIPartialGuild;
+	/**
+	 * The channel that this webhook is following (returned for Channel Follower Webhooks)
+	 */
 	source_channel?: APIPartialChannel;
+	/**
+	 * The url used for executing the webhook (returned by the [webhooks](#DOCS_TOPICS_OAUTH2/webhooks) OAuth2 flow)
```

Comment:
Does this even link?

Reply:
No. Fixed in https://github.com/discordjs/discord-api-types/pull/110/commits/e55f73a2a45af60993e22fb46bd90f563af29a4e